barcodes_gs1_label/views/options_config_views.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+     Frank Cespedes <fcespedes@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <record id="barcodes_gs1_label_options_config_form_view" model="ir.ui.view">
+        <field name="name">barcodes_gs1_label_options_config_form_view</field>
+        <field name="model">barcodes.gs1.label.options.config</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <label for="name" class="oe_edit_only" />
+                    <h1><field name="name" /></h1>
+                    <group>
+                        <group col="2">
+                            <field name="default" />
+                        </group>
+                        <group>
+                            <field name="company_id" />
+                        </group>
+                    </group>
+                    <group>
+                        <group col="2">
+                            <field name="barcode_type" />
+                            <field name="format_id" />
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="barcodes_gs1_label_options_config_tree_view" model="ir.ui.view">
+        <field name="name">barcodes_gs1_label_options_config_tree_view</field>
+        <field name="model">barcodes.gs1.label.options.config</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="barcode_type" />
+                <field name="format_id" />
+                <field name="default" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="barcodes_gs1_label_options_config_action" model="ir.actions.act_window">
+        <field name="name">Label Configuration</field>
+        <field name="res_model">barcodes.gs1.label.options.config</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <menuitem
+        id="menu_barcodes_gs1_label_options"
+        name="Barcodes Options Config."
+        parent="stock.menu_stock_config_settings"
+        groups="stock.group_stock_manager"
+        sequence="10"
+    />
```

Comment:
This is shared by config and format, it should be in another xml filem check OCA module example how to do that, maybe thet putting all menutiems in a separate xml files?